apps/client/src/intros/DakkapelIntro.js

```diff
@@ -1,26 +1,49 @@
-import { Heading } from "@datapunt/asc-ui";
+import { Heading, OrderedList, Paragraph } from "@datapunt/asc-ui";
```

Comment:
## OrderedList needs to be imported from /atoms (in all Intro files) to change this:

![image](https://user-images.githubusercontent.com/7781865/91059732-6114f600-e62a-11ea-8295-4f5d680301cb.png)

## into this:

![image](https://user-images.githubusercontent.com/7781865/91059754-67a36d80-e62a-11ea-9bdd-7f82b3796c1c.png)